src/pages/collaboration/community.astro

```diff
@@ -27,27 +52,66 @@ const page: IStandardPageMetadata = {
 <Layout title={page.title}>
   <QuickLinks links={page.section} slot="quick-links" />
   <Section {...page.section[0]}>
+  UGRC is honored to be a part of and participate in many user groups, councils, committees, associations, and communication channels. Join us in one or more of these groups to share and learn together through presentations, networking, and formal and casual communication. We look forward to meeting and working with you!
+  </Section>
+  <Section {...page.section[1]}>
     <Image
       slot="image"
       src={communityPhoto}
       loading="eager"
       alt="people together in a room"
       class="w-full max-w-[400px] rounded-2xl"
     />
-    Utah is home to a variety of user groups that cover the state and include, federal, state, and local agencies as well
-    as other interested partners.
+    Utah’s GIS user groups are valuable venues to obtain the latest information on technology, data, and projects. User groups are formed and managed by GIS practitioners in various regions throughout the state. They are great for meeting new Utah GIS community members, and connecting with longtime colleagues and friends. UGRC staff attend several user group meetings each year, when possible.
   </Section>
-  <Section {...page.section[1]}>
-    <ul>
-      <li>Utah Geographic Information Council (UGIC)</li>
-      <li>Utah Council of Land Surveyors (UCLT)</li>
-      <li>Northern Utah Geographic Information Systems group (NUGIS)</li>
-      <li>Central Utah Group (CUG)</li>
-      <li>Canyon Country Partnership (CCP)</li>
-      <li>Utah Valley GIS Users Group</li>
-      <li>G5 - Southwestern Utah counties</li>
-      <li>Uintah Basin User Group (UBUG)</li>
-      <li>Salt Lake User Group (SLUG)</li>
-    </ul>
+  <Section {...page.section[2]}>
+    <Image
+      slot="image"
+      src={communityPhoto}
+      loading="eager"
```

Comment:
**quibble**: this property is only on the first image. you can remove it from every subsequent image